int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -502,12 +500,10 @@ function setCheckoutFormEvents() {
 }
 function setEventsForListeners(checkoutStage) {
 	setEdqInputSelectors(checkoutStage);
-	removeMultipleEDQSuggestion();
```

Comment:
Deleted this line because on the checkout/billing page it was removing the Edq Global Intuitive - Address-Suggestion box and by not reloading the Global Intuitive js, the page is not creating more Edq Global Intuitive - Address-Suggestion boxes; it is just creating one box to be used.